### DISCUSSION TITLE
peak to gene

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#1305] [Questions / Documentation] [answered]
I have a question regarding the peak to gene correlation analysis.
Based on my understanding it is possible to find a peak that is correlated a gene, but the TFBS in that peak has no fragment aligned to, basically, it is not accessible, similar to the following case:

<img width="295" alt="Screen Shot 2022-02-22 at 10 04 52 PM" src="https://user-images.githubusercontent.com/19545198/155196661-05f65206-026b-4598-925e-962e73e27c1c.png">
 

Doesn't that make the peak to gene correlation a bit misleading? What if I want to know where in the peak(500bp) there is accessibility exactly?

**Comment by rcorces** [comment, marked as answer]
This seems like a pretty unlikely situation given that most peaks would require many fragments in order to be called a peak. That being said, a few comments:
1. a peak to gene link is really just a correlation and doesnt imply that any specific TF is actually causing a relationship
2. because of how insertions happen, the two insertions of a fragment dont necessarily need to span a TF binding site

> What if I want to know where in the peak(500bp) there is accessibility exactly?

This would be what single-site footprinting would be good for but I dont personally think scATAC-seq is a great way to get at this question.
